news/_posts/2020-06-12-product-features-methodology-at-prestashop.md

```diff
@@ -0,0 +1,109 @@
+The team calls these feedback **insights**. On Productboard, product managers can create topics to group **insights** together, this step helps form a first **expression of needs**. Sometimes the expression of needs already gives the best solution, sometimes it is still a bit early to know how it could be solved.
+
+What’s in it for the community? Well Productboard also makes it possible to share that externally.
+This is great to inform the community about what the team has identified, and also to keep collecting more information about it to improve the team’s understanding on this problem.
+**You are now able to see these expressions of needs on the PrestaShop Feedback website.**
+
+### How to go from a need to a feature?
+
+Centralizing and analyzing insights is difficult.
+Well, the next step in product development is even more difficult: prioritizing the users' needs.
+
+PrestaShop is used on 300k+ online stores around the world, and its community gathers more than 1 million people.
+There are hundreds of needs out there, waiting to be solved. So how does the team decide what to build next?
+
+The Core product team uses several criteria to evaluate the importance of a need. There is no scientific, or magical, formula. It is a list of aspects that the team reviews in light of the research and analysis previously run. Among these aspects are:
+
+- legal requirements,
+- the impact on the merchants’ businesses,
+- the impact on merchant productivity,
+- the impact on the developer experience,
+- and many more...
+
+As you can see, it is important for the team to take into account all the impacts of solving a need, and building features.
+And that is something that Productboard makes easier too.
+
+In the PrestaShop Feedback space, as a visitor, it is possible not only to add comments but also to rate the impact of an expression of needs or a feature on oneself. Three values exist: Nice-to-have, Important, and Critical.
+This rating is directly taken into account in Productboard and gives a **user score**. More about it in the second part.
+
+
+### Designing a feature
+
+Lastly, once you know what to focus on, and what you should work on next, it is actually time to design the feature, to define what to build.
+
+All the product development process at PrestaShop is summarized in the following chart.
+
+![Product development steps at PrestaShop](/assets/images/2020/05/product-development-steps.png)
+
+Designing a feature means defining what is the best solution to a problem, a need.
+There are many ways to solve a problem. But design is not about solving a problem, it is about solving a problem the best way possible and with elegance.
+
+The usual process to design and build a feature is the following:
+- User Research
+- Prototyping and design
+- Specifications
+- Development
+- QA testing
+- Release
+
+And sometimes there are several iterations in between to reach a complete feature.
+
```

Comment:
Here I would add a paragraph about our process on Github. 
Once the prototyping and design phase has been approved, we create an EPIC on Github with a description of the feature and all User stories issues needed to be done (+ bug issues when needed). In parallel we write down functional specifications on the specs repo on github with prototypes screenshots and links to the mockups. 

Once all of this is approved (by all stakeholders of the feature : PM, CM, Designer, dev, QA), we can starting the development phase by adding step by step user stories of the feature in the version board. 

Once all of the issues have been developed and merged, we can close the EPIC.

This process is the same when the development is done internally or with the help of the community. 

We do so to ensure alignment before going to development.

Reply:
```suggestion

Once the prototyping and design phases approved, we create an EPIC on GitHub including the feature description and all of the related user stories (each US being a specific issue) that need to be addressed. It can also contain a few bugs to fix in the meantime. 

In parallel, we write down functional specifications and add it to the dedicated repository with all prototypes screenshots and links to the mockups.

If all the stakeholders of the feature (PM, CM, designer, developer, and QA engineer) agree on it, the development phase starts. Step by step, the user stories reach the version kanban. When all issues are developed and their pull requests merged, we close the EPIC.

This process stays the same, whether the development is done internally or with the help of the community. We do so to ensure alignment before the development phase.
```

Reply:
@colinegin, it it okay for you?